adafruit_imageload/bmp/__init__.py

```diff
@@ -32,25 +32,33 @@
 __version__ = "0.0.0-auto.0"
 __repo__ = "https://github.com/adafruit/Adafruit_CircuitPython_ImageLoad.git"
 
-def load(f, *, bitmap=None, palette=None):
-    f.seek(10)
-    data_start = int.from_bytes(f.read(4), 'little')
+def load(file, *, bitmap=None, palette=None):
+    """Loads a bmp image from the open ``file``.
+
+       Returns tuple of bitmap object and palette object.
+
+       :param object bitmap: Type to store bitmap data. Must have API similar to `displayio.Bitmap`.
+         Will be skipped if None
+       :param object palette: Type to store the palette. Must have API similar to
+         `displayio.Palette`. Will be skipped if None"""
+    file.seek(10)
+    data_start = int.from_bytes(file.read(4), 'little')
     # f.seek(14)
-    # bmp_header_length = int.from_bytes(f.read(4), 'little')
+    # bmp_header_length = int.from_bytes(file.read(4), 'little')
     # print(bmp_header_length)
-    f.seek(18)
-    width = int.from_bytes(f.read(4), 'little')
-    height = int.from_bytes(f.read(4), 'little')
-    f.seek(28)
-    color_depth = int.from_bytes(f.read(2), 'little')
-    f.seek(46)
-    colors = int.from_bytes(f.read(4), 'little')
-
-    compute_palette = False
+    file.seek(18)
+    width = int.from_bytes(file.read(4), 'little')
+    height = int.from_bytes(file.read(4), 'little')
+    file.seek(28)
+    color_depth = int.from_bytes(file.read(2), 'little')
+    file.seek(46)
+    colors = int.from_bytes(file.read(4), 'little')
+
```

Comment:
These values all look correct to the [spec.](https://en.wikipedia.org/wiki/BMP_file_format)
Suggestion (no change needed):
It might be easier to maintain and check this section if the seek values are given in hex, so they match up with the documentation.

```
file.seek(0x12)  # Width of the bitmap in pixels
file.seek(0x1C)  # Number of bits per pixel